src/utils.js

```diff
@@ -1,5 +1,39 @@
-const fetchWeather = cityName => {
-  const API_KEY = "<INSERT_API_KEY>";
-  // TODO: fetch weather forecast from endpoint
-  // from https://openweathermap.org/api
+const fetchWeather = async cityName => {
+  const API_KEY = 'd29e9cce44012ae03806fcd9edc39a4e';
```

Comment:
How might you handle keeping API keys secret in these JS projects?

Reply:
I created an .env file that stores the open weather API key as REACT_APP_OPEN_WEATHER_API_KEY. We then call it in our fetch function as process.env.REACT_APP_OPEN_WEATHER_API_KEY. I left out .env in my .gitignore file so you could see exactly how I wrote it, but of course, I'd always hide my API keys. I learned that rookie mistake early on! $60 down from Rapid API later...

Reply:
Thanks for walking me through that approach! It makes sense to me. Haha, I pushed up some credentials the other day too 🙈. One of those things you learn the hard way!